content/features/introductionToFeatures/chap2/combine.md

```diff
@@ -36,4 +36,4 @@ At this stage it is important to note that the second parameter being true dispo
 
 ![house 3](/img/getstarted/house3.png)
 
-Before considering how to make multiple copies of our house we will first: find out the basics of [exporting](/features/featuresDeepDive/Exporters/glTFExporter) our models; how to [import](/features/featuresDeepDive/importers) models made with Babylon.js or other software; and how to display your scene or models on your own website.
\ No newline at end of file
+Before considering how to make multiple copies of our house we will first: find out the basics of [exporting](/features/featuresDeepDive/Exporters/glTFExporter) our models; how to [import](/features/featuresDeepDive/importers/glTF) models made with Babylon.js or other software; and how to display your scene or models on your own [website](../chap2/app2).
```

Comment:
I made both the export and import link to point to the glTF section, instead of each one pointing to different things.

Reply:
A small ask - would be great to have an absolute path instead of relative for the `app2` page

Reply:
Updated on latest iteration.